components/applications-service/pkg/storage/postgres/db.go

```diff
@@ -144,13 +133,6 @@ type serviceGroup struct {
 	DeploymentID int32     `db:"deployment_id"`
 	CreatedAt    time.Time `db:"-"`
 	UpdatedAt    time.Time `db:"-"`
-
-	// (internal) use it to know if the service-group needs an update or not
-	needUpdate bool `db:"-"`
-}
-
-func (sg *serviceGroup) NeedUpdate() bool {
-	return sg.needUpdate
```

Comment:
Service group and deployment shouldn't be changed ever, we should just move services between them.